packages/behandling-pleiepenger/src/components/PleiepengerFakta.tsx

```diff
@@ -12,11 +12,12 @@ import {
 } from '@k9-sak-web/types';
 import ac from '@fpsak-frontend/kodeverk/src/aksjonspunktCodes';
 import { LoadingPanel } from '@fpsak-frontend/shared-components';
-import { RestApiState } from '@k9-sak-web/rest-api-hooks';
+import { RestApiState, useRestApiErrorDispatcher } from '@k9-sak-web/rest-api-hooks';
 
 import faktaPanelDefinisjoner from '../panelDefinisjoner/faktaPleiepengerPanelDefinisjoner';
 import FetchedData from '../types/fetchedDataTsType';
 import { restApiPleiepengerHooks, PleiepengerBehandlingApiKeys } from '../data/pleiepengerBehandlingApi';
+import ErrorBoundary from '../../../sak-app/src/app/ErrorBoundary';
```

Comment:
Burde nok heller bruke import ErrorBoundary from '@k9-sak-web/sak-app/src/app/ErrorBoundary'; isteden for absolut path i imports. Men sen er jo kanskje spørsmålet om ErrorBoundary komponentet burde tas ut av sak-app pakken og heller ligge som en felles komponent om den skal brukes i andre pakker. Usikker der, kanskje @erlendev har noen tanker?

Reply:
kan fikse pathen 👍  Tenker ErrorBoundary er noe som burde brukes flittig i hvert fall, i main lokalt nå er det sånn at en bug i beregning gjør at hele sidemenyen og prosess-stegene på toppen forsvinner. Usikker på hvor den eventuelt skal ligge da. Er det typisk noe man putter i k9-frontend-modules?

Reply:
Ja, mye mulig den burde levd et annet sted, men ville uansett tenkt at det er lurt å ta det på siden.

Dessuten virker det som `ErrorBoundary` ikke helt gir mening å skille ut as is, i og med den har kobling til et Sentry-API. Så  implementasjonen bør kanskje endres litt på før man gjør det også?

Reply:
kan vi merge nå, hvis den ikke skal flyttes?

Reply:
Lar @erlendev merge denne, hvis det er sånn at dette er noe spesifikt for pleiepenger som ikke skal ligge rundt faktapanelene i alle behandlinger.

Reply:
stemmer, det var flere av disse ja. Jeg kan legge det til på samtlige, for det ser ut som feilhåndtering mangler på alle

Reply:
Nå har jeg lagt til på 

- Frisinn
- Omsorgpenger
- Pleiepenger
- Tilbakekreving
- Unntak
- Utvidet rett

Kunne ikke se noen faktapaneler på resten av behandlingene

Reply:
Wrappet ErrorBoundary rundt alle faktapanelene jeg fant (totalt 6 forskjellige). 

Forsøkte også legge de direkte i SideMenuWrapper, men da fører det til at man ikke kan klikke seg på en av de andre panelene etter at ErrorBoundary har catchet en feil, fordi den ikke kan vite om ting kan rerendres. Men når den legges på nivået under SideMenuWrapper vil ErrorBoundary med eventuell feil unmountes når man velger å vise et annet panel, og da rendres neste panel uten problemer. 
Derfor tror jeg ikke at jeg får gjort noe med at SonarCloud klager på duplikasjon

Reply:
@magnus-bergman ser det OK ut for deg?